TipCatDotNet.Api/Services/HospitalityFacilities/MemberService.cs

```diff
@@ -285,10 +294,18 @@ Task<Result<int>> AddMember()
         }
 
 
-        private async Task<Result<int>> AssignMemberCode(int memberId, CancellationToken cancellationToken)
+        private async Task<Result<int>> AssignMemberCode(int memberId, CancellationToken cancellationToken = default)
```

Comment:
Let the default applies only in public methods to avoid usage of different tokens.